pages/components/Typography/typography.css

```diff
@@ -0,0 +1,48 @@
+@import url(/css/main.css);
+
+.typography-preview-container {
+    display: flex;
+    flex-direction: column;
+    height: fit-content;
+    border: 1px solid var(--secondary-faded-color);
+    border-radius: 8px;
+    padding: 1rem;
+    justify-content: space-evenly;
+    gap: 1rem;
+}
+
+/* Paragraph Text Styling */
+.para-lg{
+    font-size: 1.5rem;
+}
+.para-rg{
+    font-size: 1rem;
+}
+.para-md{
+    font-size: 0.8rem;
+}
+.para-sm{
```

Comment:
naming these as font-size-sm, font-size-md would mean better I guess because naming this as para would mean these can be used only with paragraph tag.

Reply:
Yep, those are different size variants of the paragraph text only.